nakadi/swagger.yaml

```diff
@@ -227,6 +216,10 @@ paths:
         - name: event
           in: body
           description: Even being sent.
+          schema:
```

Comment:
This will make connexion validate the request body and it will fail as it is not a JSON